tinyagentos/routes/routines.py

```
@@ -0,0 +1,167 @@
+        return project_or_err
+    store = request.app.state.routine_store
+    return {"items": await store.list_routines(project_id)}
+
+
+@router.patch("/api/projects/{project_id}/routines/{rid}")
+async def update_routine(
+    project_id: str,
+    rid: str,
+    payload: UpdateRoutineIn,
+    request: Request,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)